### PR TITLE
tools: improve flow update

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Issue Number: Ref #7703.

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Manual test

After this PR:

Screenshot 2024-02-02 at 17 39 31

### Release note

```release-note
None.
```

### CODE REVIEW

File: tools/pd-heartbeat-bench/main.go [hunk 1]
```diff
@@ -46,9 +47,12 @@ import (
 )
 
 const (
-	bytesUnit            = 8 * units.MiB
-	keysUint             = 8 * units.KiB
-	queryUnit            = 1 * units.KiB
+	bytesUnit            = 128
```
Comment (lhy1024) [inline tools/pd-heartbeat-bench/main.go:50]: When we use it, we need to note that item of tikv heartbeat is the cumulative, which is used for hotspot determination is the rate.

For example, if the hotKeysUint is 256, the calculated rate is 25.6 for reads and 4.2 for writes, which is lower than the threshold for the hot side, so it won't be treated as a hot region.

File: server/api/operator.go [hunk 2]
```diff
@@ -101,6 +101,21 @@ func (h *operatorHandler) GetOperators(w http.ResponseWriter, r *http.Request) {
...
 }
 
+// @Tags     operator
+// @Summary  Cancel all pending operators.
+// @Produce  json
+// @Success  200  {string}  string  "All pending operators are canceled."
+// @Failure  500  {string}  string  "PD server failed to proceed the request."
+// @Router   /operators [delete]
+func (h *operatorHandler) DeleteOperators(w http.ResponseWriter, r *http.Request) {
+	if err := h.RemoveOperators(); err != nil {
```
Comment (CabinfeverB) [inline server/api/operator.go:111]: also use reason `operator.AdminStop`?

**Review by CabinfeverB — COMMENTED** [review]
Add some tests for API?

File: server/api/operator.go (hunk 2 above)
Comment (rleungx) [reply server/api/operator.go:111]: Any better idea?

**Review by CabinfeverB — APPROVED** [review]

**Pull request merged**